### ISSUE TITLE
Strange problem - Swal only showed with activated debugger

### ISSUE THREAD

**Issue by FredyWenger** [sweetalert2/sweetalert2#2040]
Note: Just downloaded the newest version (v9.17.1)
I have various (including complex) Swal's on my (asp.net core) page.
Some triggered directly in a view, some triggered in site.js
E.g. I have a similar like the code below, at the start of a huge view (please wait - data are loaded) and close it in $(document).ready...
All no problem.

If the data are stored (over Ajax) , I call a function in site.js from the view and show Swal's with the result (data stored / error).
This also works as expected.
Now, I want to show (additionally) an interim message ("Data are stored, please wait) with the swal below that should be showed, until the Swal to the ajax call is showed.

Problem:
- If I run the app from server or in localhost (without debugger), the swal is **not** showed.
- If I set a breakpoint in the (GC-) debugger (code halted), the message **is** showed 
- Note: The message is also not showed, if I reduce the bandwidth massive (so that the message **has** to be showed in any case)

Code:
`Swal.fire({
      icon: 'info',
      title: 'Die Daten werden nun gespeichert...',
      text: 'Bitte etwas Geduld...',
      showConfirmButton: false,
      onBeforeOpen: () => { Swal.showLoading() }
    });`

Thanks for a reply

**Comment by limonte** [comment]
Please follow the issue template when reporting issues.

![image](https://user-images.githubusercontent.com/6059356/66860984-cc1b8800-ef96-11e9-8f5f-107553d403bf.png)

**Comment by FredyWenger** [comment]
I have described anything I can in my posting.
The app is very complex, so I'm not able to provide a "working sample" out of a very complex construct.
As I wrote, Swal works without issues - expect the described issue.
if I place the posted code at another place, the Swal works.
If I activate the debugger (with stop),  it also works.
So.. it seems to be some timing issue, but I have tried anything in my mind to solve it without success.
I think, you would be able to give me some hints (also without a "live demo" or a "working sample" (whereby the posted code IS a "working sample", but not in context with the rest of the (complex) app.
